support-frontend/assets/helpers/subscriptionsForms/__tests__/subscriptionCheckoutReducerTest.ts

```diff
@@ -1,28 +1,50 @@
 // ----- Imports ----- //
+
 import {
 	setFormErrors,
 	setStage,
 } from 'helpers/subscriptionsForms/formActions';
 import { createCheckoutReducer } from 'helpers/subscriptionsForms/subscriptionCheckoutReducer';
+import type { FormField, Stage } from '../formFields';
+import type { FormError } from '../validation';
 
 jest.mock('ophan', () => () => ({}));
+
 // ----- Tests ----- //
+
 describe('Subscription Checkout Reducer', () => {
-	global.guardian = {
-		productPrices: null,
-	};
 	it('should handle SET_STAGE to "thankyou"', () => {
 		const stage: Stage = 'thankyou';
-		const action = setStage(stage);
-		const newState = createCheckoutReducer('GB')(undefined, action);
+		const action = setStage(stage, 'DigitalPack', undefined);
+
+		const newState = createCheckoutReducer(
+			'GB',
+			'DigitalPack',
+			'Annual',
+			null,
+			null,
+			null,
+		)(undefined, action);
```

Comment:
Not quite sure how we ended up with these functions being called with so many missing arguments 😂